### PR TITLE
Consolidate menu entries

### LINKED ISSUE #172390: Consolidate Online Service Settings and Telemetry Settings into one menu action
Follow up from https://github.com/microsoft/vscode-internalbacklog/issues/3401

We should consolidate "Online Service Settings" and "Telemetry Settings" into one menu action.
Ideally it will be called "Telemetry and Online Service Settings". It would just do the current Online Service Settings, but ideally the telemetry one would be closer to the top. So it does not appear hidden near the bottom. What do you think?

fyi @sandy081

### PR DESCRIPTION
**Pull request by lramos15**
Fixes https://github.com/microsoft/vscode/issues/172390

I don't love how wordy the wording is. I think we should just leave it as online services

### CODE REVIEW

**Comment by isidorn** [comment]
@lramos15 thanks for doing this.
Do we have a screenshot of the new menu - then it would be easier for me to figure out if the wording is too wordy?
Also if we call it "Online Services" do you have concerns that some users might believe that we are on purpose trying to hide telemetry settings?

**Comment by lramos15** [comment]
image
I don't think people will think we're hiding it. It now shows as the top entry in the online services tag and I believe telemetry conscious people will know where to look to find it as telemetry notices and information have been sprinkled all across the product

**Comment by isidorn** [comment]
@lramos15 thanks for the screen shot.
I am fine with whatever label you think works best. I was just slightly worried.
How about the following title that is shorter:
"Online Services Settings (Telemetry)"

**Comment by lramos15** [comment]
> "Online Services Settings (Telemetry)"

My worry here is this makes me associate all online services as telemetry related which isn't the case. I say let's stick with just `Online Services Settings` and if users complain we can revisit with their suggestions. I noticed we don't have telemetry in the command palette, maybe we could surface an entry there.

**Comment by isidorn** [comment]
That works for me. We can start with whatever you think is best.

**Review by isidorn — APPROVED** [review]

**Comment by isidorn** [comment]
Approved ✅
